godel_simple_gui/include/godel_simple_gui/blending_widget.h

```diff
@@ -27,6 +39,12 @@ class BlendingWidget : public QWidget
   void setText(const std::string& txt);
   void appendText(const std::string& txt);
   void setButtonsEnabled(bool enabled);
+  void showStatusWindow();
+  void showPlanListWidget();
+  void clearPlanList();
+  void addPlans(std::vector<std::string>& plan_names);
```

Comment:
Is the **plan_names** argument modified by the method?  If not then it should be a const argument.